discord/ext/menus/__init__.py

```diff
@@ -443,19 +509,23 @@ def remove_button(self, emoji, *, react=False):
 
         if react:
             if self.__tasks:
+
                 async def wrapped():
                     # Remove the reaction from being processable
                     # Removing it from the cache first makes it so the check
                     # doesn't get triggered.
+                    emoji = cast(discord.PartialEmoji, emoji)
```

Comment:
This currently causes a name error/unbound local error to be raised

Reply:
fixed by avoiding use of `typing.cast`